data/yearly/2024.csv

```diff
@@ -32,11 +33,13 @@ northamerica-south1,Mexico,0.19,305.00
 southamerica-east1,Sāo Paulo,0.88,67.32
 southamerica-west1,Santiago,0.92,238.00
 us-central1,Iowa,0.87,412.72
+us-central2,Oklahoma,0.88,372.23

```

Comment:
us-central2 isn't a public region. it is not listed on GCP region list or at https://cloud.google.com/sustainability/region-carbon

Can you remove?

Reply:
us-central2 isn't a secret, so why not